Makefile

```diff
@@ -0,0 +1,28 @@
+
+## jinja_tornado Makefile
+
+default: build
+
+clean:
+	rm -rf ./jinja_tornado*.egg-info
+	rm -rf ./build/
+	rm -rf ./dist/
+
+install-build-requirements:
+	#@pip install -r ./requirements.txt
+	pip install wheel twine
+
+build: clean
+	python setup.py build
+
+dist: build
+	python setup.py sdist bdist_wheel
+
+register:
+	python setup.py register
+
+release:
+	@# $(MAKE) register
+	twine upload dist/*
```

Comment:
https://pypi.python.org/pypi/twine does the right thing w/ pypi and HTTPS

- AFAIU the  setup.cfg/pypi.cfg/.config/pip/pip.conf  index url should be ``https`` before/for ``setup.py register``